### DISCUSSION TITLE
Blog feature suggestions

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#4787] [Ideas]
After waiting for a bit to see if some features get added after the initial release am I now taking the time to suggest some features for the blog plugin to consider.
Some of these features may sound familiar to you (Squidfunk) and that may be because I suggested them in the original issue, where you requested feedback and ideas for the plugin.

### "Suggestions" option
This would be a metadata option, where you can set a list of blog-posts to recommend at the bottom of the page.
The entries would be the path to the file(s) relative to the blog directory.

Ideally would MkDocs-Material pull data from these posts such as title, description/excerpt and social cards.
The inspiration for this idea cam from the Blog of the originsrealms server, where other posts were recommended at the bottom with the blog's title image (or what you call that) being displayed.
Example:
![grafik](https://user-images.githubusercontent.com/11576465/208932218-7d22501d-7494-4900-acc5-7387cb02e097.png)

This would also allow to more easily link posts together, by recommending follow-up posts to continue reading, or other posts related to your current one.

### Multiple Blogs
There may be a situation, where you would like to have separate directories for your blogs, to further separate them.

As an example, imagine having a `updates` subpage where you post about updates towards your projects and a `general` subpage reserved for other, random blog posts not fitting the first one.

With the current setup is there no way to keep them differenciated. You only have a single directory to set and while you could have something like `blog/updates` and `blog/general` is that not an ideal setup as things like the Category index may not reflect this that well (You may want to have different categories per sub-page).

So, if technically possible would it be welcome if you could define multiple directories for the blogs, including own categories that aren't part of the global one (which perhaps can/should be considered too?)

This would probably be one of the more aggressive changes as it would need breaking configuration changes in some way, so I understand if this is something you wouldn't want to have implemented.

---

Either way, those are the ideas I had on my mind for quite a bit and I hope they are somewhat useful.

**Comment by squidfunk** [comment]
Thanks for your suggestions. I will consider them when continuing work on the blog plugin.

Regarding __multiple blogs__ - this is already fully supported. Everything you've mentioned works out of the box. You can have as many instances of the `blog` plugin as you like. For further info, see https://github.com/squidfunk/mkdocs-material/discussions/4727.

**Reply by Andre601** [reply]
Oh, that's quite handy. Didn't know Python Markdown allows multiple instances of the same plugin to be defined.

**Reply by squidfunk** [reply]
Yes, we're working on a new [example configurations](https://github.com/mkdocs-material/examples) repository, and multiple blogs is definitely an interesting case 😊
